utils/nvme-cli/patches/020-usl-fix-build-due-to-undeclared-locale.patch

```diff
@@ -0,0 +1,44 @@
+I don't think musl has the ability to read the locale of the build system
+so in order to fix the following build error, just set it with this commit.
```

Comment:
May I know, if this patch is included in upstream and if not, could you please upstream it?

Reply:
Please see @hnyman's comment. I backported two of their commits and our package is good.